README.md

```diff
@@ -1,6 +1,6 @@
 # CrowdStrike Falcon Helm Chart and Helm Operator
 
-[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm) [![Docker Repository on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator/status "Docker Repository on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator)
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm) [![Helm Operator on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator/status "Helm Operator on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator) [![Helm OLM Bundle on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator-bundle/status "Helm OLM Bundle on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator-bundle)
```

Comment:
```suggestion
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm) [![Helm Operator on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator/status "Helm Operator on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator) [![Helm OLM Bundle on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator-bundle/status "Helm Operator Lifecycle Manager (OLM) Bundle on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator-bundle)
```

Reply:
should we expand ``OLM``?

Reply:
Don't know..... I guess we can and change it later if it doesn't work or look right.

Reply:
I updated it in one part but not the other as I believe that we could get into trouble around double parenthesis.